lib/jaeger/client/udp_sender.rb

```diff
@@ -67,6 +68,14 @@ def emit_batch(thrift_spans)
         )
 
         @client.emitBatch(batch)
+      rescue StandardError => error
+        log_error(error, batch)
+      end
+
+      def log_error(error, failed_batch)
+        return unless @logger
+
+        @logger.error("[jaeger-client] Failure while sending batch: #{failed_batch.inspect} with error: #{error}")
```

Comment:
Minor comment -  log formatting, here prefixing `[jaeger-client]`, should be a behavior of the `logger` itself.
A quick search for `[jaeger-client]` didn't reveal this prefix in use anywhere else in the lib - we should we consider if we want to introduce it buried in this class here.

Reply:
I also agree, please remove the prefix.

My one other concern is doing inspect on `failed_batch`. This can be a quite huge if the service has a lot of traffic and/or flush interval is high which may cause unnecessary IO and may slow down the service. Maybe the batch should be logged as a debug statement as a separate log statement. What do you think?

Reply:
Yes, I've already noticed that those inspects can get really big, however it's helpful in debugging.

Reply:
> and also add that prefix in logger

The app client can do it itself and inject it to this library. No need to add any prefixes here. E.g. we're using https://github.com/salemove/lenjador and add these kind of things as a metadata instead of prefixes.